cf-custom-resources/lib/env-controller.js

```diff
@@ -132,27 +137,27 @@ const controlEnv = async function (
     } catch (err) {
       if (
         !err.message.match(
-          /^Stack.*is in UPDATE_IN_PROGRESS state and can not be updated/
+            /^Stack.*is in UPDATE_IN_PROGRESS state and can not be updated/
         )
       ) {
         throw err;
       }
       // If the other workload is updating the env stack, wait until update completes.
       await cfn
+          .waitFor("stackUpdateComplete", {
+            StackName: stackName,
+            $waiter: updateStackWaiter,
+          })
+          .promise();
+      continue;
+    }
```

Comment:
Why do we make the change to place stack status checking here?

Reply:
I think it's the same as before but for whatever reason the diff is rendered differently

Reply:
Yeah you are right